### PR TITLE
fix new info menu

### PR DESCRIPTION
**Pull request by DanielPalafox**
https://trello.com/c/OIlbX1qu/6829-registry-changes-needed-for-info-site-migration-5

### CODE REVIEW

File: src/app/layout/header/header.component.scss-theme.scss [hunk 1]
```diff
@@ -15,6 +15,29 @@
...
+    color: mat-color($primary, '500-contrast');
+    background-color: mat-color($primary);
+
+    .menu-container {
+      .row {
+        button {
+          &.active {
+            background-color: mat-color($background, grey);
+            color: mat-contrast($accent, 500);
+            border-radius: 0;
```
Comment (leomendoza123) [inline src/app/layout/header/header.component.scss-theme.scss:27]: the theme file should only contain color-related styles

File: src/app/layout/header/header.component.scss-theme.scss [hunk 2]
```diff
@@ -45,7 +68,37 @@
         background-color: mat-color($background, 'card');
       }
     }
+
     .top-bar {
+      background-color: mat-color($primary, 700);
+      .menu-container {
+        .row {
+          button {
```
Comment (leomendoza123) [inline src/app/layout/header/header.component.scss-theme.scss:76]: i believe this can be removed

File: src/environments/environment.local.ts [hunk 3]
```diff
@@ -5,7 +5,7 @@ export const environment = {
   API_WEB: '//dev.orcid.org/',
   BASE_URL: '//dev.orcid.org/',
   BLOG_NEWS: 'https://localhost:8888/about/news',
-  INFO_SITE: 'https://info.orcid.org',
```
Comment (leomendoza123) [inline src/environments/environment.local.ts:8]: let's add this fix to the other environments as well, like sandbox

File: src/app/layout/header/menu.ts [hunk 4]
```diff
@@ -1,9 +1,341 @@
...
+    //     },
+    //   },
+    // ],
+  },
+  {
+    id: 'organizations',
+    route: 'organizations',
+    label: $localize`:@@public-layout.for_organizations:For Organizations`,
+    requirements: {
+      togglz: { NEW_INFO_SITE: 'false' },
```
Comment (leomendoza123) [inline src/app/layout/header/menu.ts:161]: nice to see you are taking advantage of this old requirements feature 💯

**Comment by DanielPalafox** [comment]
[Registry changes needed for info site migration (5)](https://trello.com/c/OIlbX1qu/6829-registry-changes-needed-for-info-site-migration-5)

File: src/app/layout/header/header.component.scss-theme.scss (hunk 1 above)
Comment (DanielPalafox) [reply src/app/layout/header/header.component.scss-theme.scss:27]: Ok. I will remove it.

File: src/environments/environment.local.ts (hunk 3 above)
Comment (DanielPalafox) [reply src/environments/environment.local.ts:8]: Ok

File: src/app/layout/header/header.component.scss-theme.scss (hunk 2 above)
Comment (DanielPalafox) [reply src/app/layout/header/header.component.scss-theme.scss:76]: Ok

**Pull request merged**